lib/src/model/constructor.dart

```diff
@@ -33,7 +33,7 @@ class Constructor extends ModelElement
 
   @override
   Container get enclosingElement =>
-      modelBuilder.from(element.enclosingElement, library) as Container;
+      packageGraph.getModelFor(element.enclosingElement, library) as Container;
```

Comment:
This and other files -- why do we sometimes call `getModelFor` or `getModelForPropertyInducingElement` with `packageGraph` and other times we don't? Can we just call the helper itself (`getModelFor(element.enclosingElement, library) as Container`)?

Reply:
Thanks for pointing this out!

This is a relic from halfway through this refactoring, before I moved the convenience methods to `Nameable`. I've just pushed a commit that simplifies a dozen call sites.